canary/webrtc-c/src/Include.h

```diff
@@ -97,6 +98,7 @@
 #define KVS_METRICS_INVOCATION_PERIOD        (5 * HUNDREDS_OF_NANOS_IN_A_SECOND)
 #define CANARY_METADATA_SIZE                 (SIZEOF(UINT64) + SIZEOF(UINT32) + SIZEOF(UINT32))
 
+#define STORAGE_CANARY_FIRST_FRAME_TS_PATH      "../firstFrameSentTimeStamp.txt"
```

Comment:
The file name should be an env. The same file name is being used in consumer and it is very easy to miss changing the name on both ends. An env solves this since it will be commonly used by consumer and webrtc.

Something like: `#define STORAGE_FIRST_FRAME_TS_FILE_PATH_ENV STORAGE_FIRST_FRAME_TS_FILE_PATH`

Usage would be: `export STORAGE_FIRST_FRAME_TS_FILE_PATH=<path-to-file>`